modules/static/gogo/protobuf/initref

```diff
@@ -1 +1 @@
-f6b4bb7b2dde1736b809b3da996ed72f278e9be9
+v1.3.2
```

Comment:
This is the latest release: https://github.com/gogo/protobuf/releases

The closest release to the original initref commit would have been:

```
➜  protobuf git:(f6b4bb7b) git describe --all f6b4bb7b2dde1736b809b3da996ed72f278e9be9
tags/v0.3-140-gf6b4bb7b
```